src/qt/guiutil.cpp

```diff
@@ -1,4 +1,5 @@
 // Copyright (c) 2011-2019 The Bitcoin Core developers
+// Copyright (c) 2019-2020 The Bitcoin Core developers
```

Comment:
Change to Veil developers.

Reply:
oops.  nice catch.  joys of distractions :)  All set.